api.php

```diff
@@ -373,6 +374,64 @@ public function run($userID, $permissionIsExplicit) {
 	}
 }
 
+/**
+ * API entry game/overview
+ * 
+ * This should be cleaned up. 
+ */
+class GetGameOverview extends ApiEntry {
+	public function __construct() {
+		parent::__construct('game/overview', 'GET', 'getStateOfAllGames', array('gameID'));
+	}
+
+	private function getMemberData( $members ){
+		return array_map( function( $member ){
+			return [
+				'id' => $member->id,
+				'userID' => $member->userID,
+				'countryID' => $member->countryID,
+				'country' => $member->country,
+				'online' => $member->online,
```

Comment:
if the order doesn't matter, we should alpha-order these for ease of use

Reply:
Agree, will update this.